.gitignore

```diff
@@ -127,3 +130,12 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# data
+photo
+text
+video
+
+# extra
+accounts.txt
+yt-dlp.exe
```

Comment:
Small additions to ensure that downloaded content and the YT-DLP binary aren't accidentally pulled into commits.